### DISCUSSION TITLE
(Installation) only got to the first step and don't know what to do next

### DISCUSSION THREAD

**Discussion by octagon7610** [Kyrluckechuck/TFT-Bot#239] [Q&A]
i download Python 3.11.0
Navigate to my install directory (which installs, Python? or the TFT-Bot-main file.)
Then I typed "cd" in Python 3.11.0. I don't know what is it but you use grey highlight on things in "readme" that seem considered as code. 
so I open Python 3.11.0 and type in "cd" 
It reply "SyntaxError: invalid syntax"

I'm not a programmer... so this is the furthest i can do u_u

**Comment by Kyrluckechuck** [comment]
I can't break down the steps too much more, but the install directory is referring to wherever you have downloaded the code from this repo to (whether the lastest release or the latest `main` branch)
If you haven't yet, you can download the latest `main` from this URL: https://github.com/Kyrluckechuck/TFT-Bot/archive/refs/heads/main.zip
You will then need to unzip it, and navigate to the folder that it was output to using the Windows Command Prompt
Here is a quick overview on how to use `cd` in Cmd: https://www.wikihow.com/Change-Directories-in-Command-Prompt
